stately-isolate/src/commonMain/kotlin/co/touchlab/stately/isolate/IsoMutableMap.kt

```diff
@@ -1,46 +1,55 @@
 package co.touchlab.stately.isolate
 
-class IsoMutableMap<K, V>() : IsolateState<MutableMap<K, V>>({ mutableMapOf() }), MutableMap<K, V> {
+class IsoMutableMap<K, V> : IsolateState<MutableMap<K, V>>({ mutableMapOf() }), MutableMap<K, V> {
+
     override val size: Int
-        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.
+        get() = runBlocking {
+            access { it.size }
+        }
 
     override fun containsKey(key: K): Boolean = runBlocking {
         access { it.containsKey(key) }
     }
 
-    override fun containsValue(value: V): Boolean {
-        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+    override fun containsValue(value: V): Boolean = runBlocking {
+        access { it.containsValue(value) }
     }
 
-    override fun get(key: K): V? {
-        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+    override fun get(key: K): V? = runBlocking {
+        access { it[key] }
     }
 
-    override fun isEmpty(): Boolean {
-        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+    override fun isEmpty(): Boolean = runBlocking {
+        access { it.isEmpty() }
     }
 
     override val entries: MutableSet<MutableMap.MutableEntry<K, V>>
-        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.
+        get() =  runBlocking {
+            access { it.toMutableMap().entries }
```

Comment:
Values from map need to be copied to prevent freezing, simplest way was to call toMutableMap.

Reply:
The expectation is that the returned `MutableSet` will interact with the original. Let's talk about this tomorrow. I think we could return an isolated MutableSet implementation that would just wrap `entries` from the original. Getting that set up will be a little tricky, but will show the value of using state this way. We can create concurrent functionality without a lot of custom code.